src/ale_interface.hpp

```diff
@@ -107,6 +107,17 @@ class ALEInterface {
   // Returns the current game screen
   const ALEScreen &getScreen();
 
+  //This method should receive an array of length width x height
+  //(generally 160 x 210 = 33,600) and then it will fill this array
+  //with the grayscale colours
+  void getScreenGrayscale(pixel_t *grayscale_output_buffer);
+
+  //This method should receive an array of length 3 x width x height
```

Comment:
The comment needs to be updated: this is [currently computed as] width x height x 3, rather than 3 x width x height.